Analysis/Tasks/PWGHF/qaTask.cxx

```diff
@@ -183,6 +184,18 @@ struct QAGlobalObservables {
 
     histograms.add("multiplicity/numberOfTracks", qafeat::MakeTitle({qafeat::TrackMultiplicity}).c_str(), o2fw::kTH1D,
                    {{nBinsNumberOfTracks, numberOfTracksRange[0], numberOfTracksRange[1]}});
+
+    // covariance histograms
+    histograms.add("Covariance/xx", "xx;Cov_{xx} [cm]", kTH1D, {{200, -0.1, 0.1}});
+    histograms.add("Covariance/xy", "xy;Cov_{xy} [cm]", kTH1D, {{200, -0.1, 0.1}});
+    histograms.add("Covariance/xz", "xz;Cov_{xz} [cm]", kTH1D, {{200, -0.1, 0.1}});
+    histograms.add("Covariance/yy", "yy;Cov_{yy} [cm]", kTH1D, {{200, -0.1, 0.1}});
+    histograms.add("Covariance/yz", "yz;Cov_{yz} [cm]", kTH1D, {{200, -0.1, 0.1}});
+    histograms.add("Covariance/zz", "zz;Cov_{zz} [cm]", kTH1D, {{200, -0.1, 0.1}});
```

Comment:
The units are `cm^{2}` not `cm`.

Reply:
True, I missed it when copying the lines, I will fix this in the next PR